signing_clients/tests/__init__.py

```diff
@@ -129,3 +134,11 @@ def test_08_unicode(self):
                                  'signing_clients/tests/test-jar-unicode-signed.jar',
                                  omit_signature_sections=False)
         self.assertEqual(str(extracted.manifest), UNICODE_MANIFEST)
+
+    def test_09_serial_number_extraction(self):
+        with open('signing_clients/tests/zigbert.test.pkcs7.der', 'r') as f:
```

Comment:
this file path is fragile and may not work with certain test runners (e.g. sometimes Nose changes directories). You can do something like: ``os.path.join(os.path.dirname(__file__), 'zigbert.test.pkcs7.der')``